native/src/routes/search/containers/SearchModalContainer.js

```diff
@@ -36,7 +42,8 @@ const mapStateToProps = (state: StateType, ownProps: OwnPropsType) => {
     categories: state.cityContent?.searchRoute?.categoriesMap || null,
     language: state.contentLanguage,
     cityCode,
-    closeModal: () => {
+    closeModal: (query: string) => {
+      sendTrackingSignal({ signal: { name: SEARCH_FINISHED_SIGNAL_NAME, query, url: null } })
```

Comment:
Would it maybe make more sense to handle this in `SearchModal` as the other tracking signal is sent there as well?

Reply:
I decided to sendSignal where the navigation action is executed, but i can also be done as you suggested. changed it.